bridges/DanbooruBridge_Fix_Simple_Html_Dom.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * This class is a monkey patch to 'extend' simplehtmldom to recognize <source>
+ * tags (HTML5) as self closing tag. This patch should be removed once
+ * simplehtmldom was fixed. This seems to be a issue with more tags:
```

Comment:
Since simplehtmldom is abandoned, I am considering to make this patch directly in vendor/simplehtmldom directory or fork simplehtmldom in RSS-Bridge/simplehtmldom. What do you think about it?

Reply:
Since it's abandoned I think moving it to `vendor/simplehtmldom` is fine.

Reply:
Maybe later we can seriously consider maintaining a fork under the rss-bridge organization. We probably have to live with this dependency a long time.

Reply:
New bridges in the future should maybe start using:
https://symfony.com/doc/current/components/css_selector.html

or any other modern DOM crawlers.

Reply:
If I remember correctly, this bug was addressed a long time ago. The monkey patch should no longer be necessary in the current version of simplehtmldom.